.azure-pipelines/client.yml

```diff
@@ -75,6 +75,26 @@ jobs:
 
       - script: |
           node common/scripts/install-run-rush.js install
+        displayName: 'Install dependencies'
+
+      - script: |
+          node common/scripts/install-run-rush.js lint
+        displayName: 'Run lint'
+
+      - task: CopyFiles@2
```

Comment:
Can we avoid this copy by passing in the output path as a command line parameter to the lint command?

Reply:
I got the creating the reports directly to artifact staging directory working with this command : `"lint": "cross-env eslint -c ../../.eslintrc.json src test samples --ext .ts -f node_modules/eslint-detailed-reporter/lib/detailed.js -o $BUILD_ARTIFACTSTAGINGDIRECTORY/service-bus-lintReport.html || exit 0",`  

But this would disrupt the local dev environment. The developers need to be aware of creating the environment variable and passing in the appropriate value before running the linter. Discussed this with @bsiegel and @scbedd , we would want to keep both the Dev and CI environments consistent.

Reply:
I agree we shouldn't use the env variable directly in the script command. For now lets stick with what you have. Thanks for investigating options.